doozer/doozerlib/rhcos.py

```
@@ -27,7 +27,15 @@ class RHCOSNotFound(Exception):
 
 
 class RHCOSBuildFinder:
-    def __init__(self, runtime, version: str, brew_arch: str = "x86_64", private: bool = False, custom: bool = False):
+    def __init__(
+        self,
+        runtime,
+        version: str,
+        brew_arch: str = "x86_64",
+        private: bool = False,
+        custom: bool = False,
+        registry_config: str = None,
```

Ruff (0.15.9)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)